src/styles/bloks/card.scss

```
@@ -0,0 +1,113 @@
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+
+  min-width: none;
+  max-width: 1024px;
+  margin: 0 auto;
+  padding-top: 50px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin: 0 50px 48px 0;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &_img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 0 40px 3px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $black-for-card;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+
+    color: $grey-for-card;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__rating {
+    margin-bottom: 24px;
+
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  .price {
```

```suggestion
  &__price {
```
fix BEM according to task description